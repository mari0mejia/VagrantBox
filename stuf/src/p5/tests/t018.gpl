on
akey
{
//Use semantic value of T_PRINT.
//Don't use line_count directly.
print
(
"This is " +
"spread across " +
"multpile lines."
)
;
}
