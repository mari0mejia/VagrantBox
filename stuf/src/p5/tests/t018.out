gpl.cpp::main()
  input file(tests/t018.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.
print[6]: This is spread across multpile lines.

User quit program. Printing the symbol table.
